compiler/damlc/tests/src/DA/Test/GenerateSimpleDalf.hs

```diff
@@ -47,7 +47,7 @@ main = do
             , dataTypeCon = TypeConName ["Choice"]
             , dataSerializable = IsSerializable True
             , dataParams = []
-            , dataCons = DataVariant [(VariantConName "Choice", TUnit)]
+            , dataCons = DataRecord [ (FieldName "choiceArg", TUnit) ]
```

Comment:
Why can’t we support variants here? I know that it’s not possible to generate such code from DAML since we always build up a record (unless you manually write `Choice` instances) but it seems fairly easy to support this. in fact, it probably works out of the box unless you validate that the type is a record which I think you don’t do at the moment.

If we don’t support it as part of this PR, that’s fine as well but in that case I would like to keep the test case as an expected failure.

Reply:
The reason to change to a record type was because I wanted to test a cross-sdk exercise using DAML. I'll make the generated DALF have 2 choices: One with a record; one with a variant.

Reply:
Sounds good :+1:

Reply:
Looks like you didn’t do this? We still seem to only have one choice for a `Record`.

Reply:
done now. 
But I'm not really convinced of the utility of creating DALF for testing which is not of the form which would result from the daml compiler.